src/factor.f90

```diff
@@ -23,8 +23,8 @@ subroutine findfactor(num, factors, nfact)
 
       ! find the factors <= sqrt(num)
       ! Cast the int as double to make sure of the correct result of sqrt
-      ! IntelLLVM got an issue with 1.0 but not with 1.d0 
-      m = int(sqrt(real(num,8)))
+      ! IntelLLVM got an issue with 1.0 but not with 1.d0
```

Comment:
I think the comment is now out of date? And `real(num, 8)` isn't portable, suggest `real(num, kind(1.0d0))` or `real(num, mytype)`

Reply:
Should be fixed in https://github.com/2decomp-fft/2decomp-fft/pull/244/commits/1b085bd668d5c8f49a7f433dfa4ae15f961d3ddd.